trs_cli/client.py

```diff
@@ -88,6 +94,81 @@ def __init__(
     #
     # Check DRS-cli repo for examples
 
+    def get_descriptor(
+        self,
+        type: str,
+        id: str,
+        version_id: str,
```

Comment:
This should be `version_id: Optional[str]` because a version ID could also be extracted from a TRS URI, if that is passed instead of a tool ID. Check the next comments and the docstring of `_get_tool_id_version_id()` for details.